lib/external_upload_helpers.rb

```diff
@@ -354,8 +357,13 @@ def validate_part_number(part_number)
 
   def debug_upload_error(err, friendly_message)
     return if !SiteSetting.enable_upload_debug_mode
-    Discourse.warn_exception(err, message: friendly_message)
-    (Rails.env.development? || Rails.env.test?) ? friendly_message : I18n.t("upload.failed")
+    Discourse.warn_exception(err, message: "[ExternalUploadError] #{friendly_message}")
+
+    if (Rails.env.development? || Rails.env.test? || is_api?)
```

Comment:
There's a new alias for this:

```suggestion
    if (Rails.env.local? || is_api?)
```

Reply:
Ooh nice one!